parent/pom.xml

```diff
@@ -2854,35 +2878,38 @@
                     <artifactId>javax.annotation-api</artifactId>
                 </dependency>
                 <dependency>
-                    <groupId>javax.xml.ws</groupId>
-                    <artifactId>jaxws-api</artifactId>
-                    <version>2.3.1</version>
-                    <exclusions>
-                        <exclusion>
-                            <groupId>javax.xml.bind</groupId>
-                            <artifactId>jaxb-api</artifactId>
-                        </exclusion>
-                    </exclusions>
+                    <groupId>jakarta.xml.ws</groupId>
```

Comment:
This guy brings jakarta.annotation-api, should we get rid of javax.annotation-api in this case?